pytorch/inference/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -157,7 +157,7 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
 
 RUN pip uninstall -y model-archiver multi-model-server \
  && pip install captum \
- && pip install torchserve==${TS_VERSION} \
+ && pip install torchserve-nightly==2022.3.23.post2 \
```

Comment:
Let's only make this change on PT 1.10 inference DLCs, for now, and make changes on other framework versions once this change goes GA.